powerhome/playbook
# [PLAY-662] Recoded separator to remove z-index issues

- Removed the need for a z-index in the CSS
- Removed the need to provide a background prop. This now works on any background (prop remains however)
- Fixed several dark mode errors in PB website + Section Separator Kit + Nav Kit

#### Screens

![Image 2023-03-07 at 10 42 54 AM](https://user-images.githubusercontent.com/863031/223497308-1f4c027e-3e9d-42e4-9c57-44a188365f28.jpg)
Left Prod | Right PR

#### Breaking Changes

No but should be well tested in Nitro. This does remove the need to pass a background to the caption. This was was added as a fix to a hack, but should be considered for depreciation in a future release.

#### Runway Ticket URL

[PLAY-662](https://nitro.powerhrg.com/runway/backlog_items/PLAY-662)

#### How to test this

View a page using the section separator, and confirm the text and the borders are displaying as intended. Test Rails & React

#### Checklist:
